test/integration/api/app-users.js

```diff
@@ -38,6 +40,7 @@ describe('api: /projects/:id/app-users', () => {
             .send(testData.instances.simple.one)
             .expect(403)))));
 
+    // eslint-disable-next-line no-unused-vars
```

Comment:
no-unused-vars is probably easy enough to fix in every case we should ideally do them rather than commit the lint exception.